config/models

```diff
@@ -3,7 +3,8 @@ User
     lastName Text
     email Text
     ident Text
-    UniqueUser ident
+    plugin Text
+    UniqueUser ident plugin
```

Comment:
I would do plugin then ident. It reads better to me and is the order of the `Creds` datatype.